scania-digital-design-system/sdds
# fix(table): refactor event names

**Describe pull-request**  
Refactored event names and updated story. 


**Solving issue**  
Fixes: [AB#3361](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3361)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Data Table -> Check all web component stories
3. Check the readme and event names to make sure they conform with convention.
4. Check if there are any events that are marked as internal now that should be external, and vice versa.
5. Check the event names. Since some event are so specific to this component I had to digress a bit from the naming convention.. 😢 

**Suggested test steps**
